### PR TITLE
docs(samples): add export to BigQuery sample and test 

### PR DESCRIPTION
**Pull request by lvvvvvf**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-asset/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X ] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

Fixes #<212> 🦕

### CODE REVIEW

File: samples/exportAssets.js [hunk 1]
```diff
@@ -50,7 +50,38 @@ async function main(dumpFilePath) {
     // Do things with with the response.
     console.log(result);
   }
+
+  async function exportAssetsBigquery() {
```
Comment (bcoe) [inline samples/exportAssets.js]: If possible, let's pull this into its own sample file `samples/exportAssetsBigquery.js`.

File: samples/exportAssets.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018, Google, LLC.
+ * Copyright 2019, Google, LLC.
```
Comment (bcoe) [inline samples/exportAssets.js]: we shouldn't change the copyright date when we update the file.

File: samples/test/sample.test.js [hunk 3]
```diff
@@ -26,22 +26,42 @@ const storage = new Storage();
...
+    await bigquery
+      .dataset(datasetName)
+      .delete({force: true})
+      .catch(console.warn);
   });
 
   it('should export assets to specified path', async () => {
     const dumpFilePath = `gs://${bucketName}/my-assets.txt`;
-    execSync(`node exportAssets ${dumpFilePath}`);
+    const dataSet = `datasets/${datasetName}`;
```
Comment (bcoe) [inline samples/test/sample.test.js]: appreciate the tests \o/

File: samples/exportAssets.js (hunk 1 above)
Comment (lvvvvvf) [reply samples/exportAssets.js]: Done

**Comment by lvvvvvf** [comment]
Pls take a look, thx

**Comment by lvvvvvf** [comment]
> it looks like the sample test is failing for the BigQuery sample:
> 
> ```
>   1) quickstart sample tests
>        should export assets to specified BigQuery table:
>      Error: Timeout of 180000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/tmpfs/src/github/nodejs-asset/samples/test/sample.test.js)
>       at process._tickCallback (internal/process/next_tick.js:68:7)
> ```
> 
> any thoughts?

Forgot to remove delete table part from gcs,  now should be fixed, Pls take a look.  Thanks

**Comment by bcoe** [comment]
@lvvvvvf sorry that this has been stale, it looks like tests are still failing (I've kicked them off again in case the next run works).

Are you able to get the samples running locally?

**Comment by lvvvvvf** [comment]
It works for me.


________________________________

**Comment by bcoe** [comment]
@lvvvvvf have you added any new resources that need to be in our testing account perhaps? Sample tests seem to be fairly consistently failing.

**Comment by bcoe** [comment]
@lvvvvvf looks like this has gotten a bit stale. If you'd still like to land, please let me know. We just need to resolve the conflicts and get the tests running 👍

**Comment by lvvvvvf** [comment]
$npm i && npm test
audited 1633 packages in 2.52s
found 14 vulnerabilities (8 low, 3 moderate, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details

> @google-cloud/asset@1.5.0 test /usr/local/google/home/lvv/nodejs-asset
> npm run cover


> @google-cloud/asset@1.5.0 cover /usr/local/google/home/lvv/nodejs-asset
> nyc --reporter=lcov mocha test/*.js && nyc report



  AssetServiceClient
    ✓ has servicePath
    ✓ has apiEndpoint
    ✓ has port
    ✓ should create a client with no options (91ms)
    ✓ should create a client with gRPC fallback
    exportAssets
      ✓ invokes exportAssets without error (48ms)
      ✓ invokes exportAssets with error (45ms)
      ✓ has longrunning decoder functions
    batchGetAssetsHistory
      ✓ invokes batchGetAssetsHistory without error (38ms)
      ✓ invokes batchGetAssetsHistory with error

  AssetServiceClient
    ✓ has servicePath
    ✓ has apiEndpoint
    ✓ has port
    ✓ should create a client with no options
    ✓ should create a client with gRPC fallback
    exportAssets
      ✓ invokes exportAssets without error (47ms)
      ✓ invokes exportAssets with error (51ms)
      ✓ has longrunning decoder functions
    batchGetAssetsHistory
      ✓ invokes batchGetAssetsHistory without error (38ms)
      ✓ invokes batchGetAssetsHistory with error (43ms)
    createFeed
      ✓ invokes createFeed without error (39ms)
      ✓ invokes createFeed with error (38ms)
    getFeed
      ✓ invokes getFeed without error
      ✓ invokes getFeed with error
    listFeeds
      ✓ invokes listFeeds without error
      ✓ invokes listFeeds with error
    updateFeed
      ✓ invokes updateFeed without error (42ms)
      ✓ invokes updateFeed with error (39ms)
    deleteFeed
      ✓ invokes deleteFeed without error
      ✓ invokes deleteFeed with error

  AssetServiceClient
    ✓ has servicePath
    ✓ has apiEndpoint
    ✓ has port
    ✓ should create a client with no options
    ✓ should create a client with gRPC fallback
    exportAssets
      ✓ invokes exportAssets without error
      ✓ invokes exportAssets with error
      ✓ has longrunning decoder functions
    batchGetAssetsHistory
      ✓ invokes batchGetAssetsHistory without error (51ms)
      ✓ invokes batchGetAssetsHistory with error
    createFeed
      ✓ invokes createFeed without error
      ✓ invokes createFeed with error
    getFeed
      ✓ invokes getFeed without error
      ✓ invokes getFeed with error
    listFeeds
      ✓ invokes listFeeds without error
      ✓ invokes listFeeds with error
    updateFeed
      ✓ invokes updateFeed without error
      ✓ invokes updateFeed with error
    deleteFeed
      ✓ invokes deleteFeed without error
      ✓ invokes deleteFeed with error


  50 passing (1s)

**Pull request closed without merging**